adopt-github-release/src/main/groovy/net/adoptopenjdk/release/UploadFiles.groovy

```diff
@@ -126,6 +132,8 @@ private OptionAccessor parseArgs(String[] args) {
                 d longOpt: 'description', type: String, args: 1, 'Release description'
                 r longOpt: 'release', 'Is a release build'
                 h longOpt: 'help', 'Show usage information'
+                s longOpt: 'server', type: String, args: 1, 'Github server (optional)'
+                o longOpt: 'org', type: String, args: 1, 'Github org (optional)'
```

Comment:
If it were me, I would have made these required but with a `defaultValue:` rather than handle the null values in `Release.sh`, but this works too - only because it removes null as a value with special meaning hidden in Release.sh.